cli/ioctl/cmd/account/accountimport.go

```diff
@@ -89,7 +89,7 @@ func newAccountByKey(name string, privateKey string) (string, error) {
 		return "", errors.New("password doesn't match")
 	}
 	ks := keystore.NewKeyStore(config.ConfigDir, keystore.StandardScryptN, keystore.StandardScryptP)
-	priKey, err := keypair.DecodePrivateKey(privateKey)
+	priKey, err := crypto.HexToECDSA(privateKey)
```

Comment:
Our goal is not to use eth crypto, right?

Reply:
will do another PR to remove eth crypto